### PR TITLE
Allow environment variable to point to OPENQA_LIBPATH

### PR DESCRIPTION
**Pull request by foursixnine**
To help with new a parser development, an environment variable comes in handy to specify where the current openQA code is

### CODE REVIEW

File: testapi.pm [hunk 1]
```diff
@@ -1699,7 +1699,7 @@ sub parse_extra_log {
         local $@;
         # We need to touch @INC as specific supported format are split
         # in different classes and dynamically loaded by OpenQA::Parser
-        local @INC = (OPENQA_LIBPATH, @INC);
+        local @INC = ($ENV{OPENQA_LIBPATH}, OPENQA_LIBPATH, @INC);
```
Comment (coolo) [inline testapi.pm]: wouldn't an empty value be troublesome?

Comment (Martchus) [reply testapi.pm]: Maybe make it `local @INC = ($ENV{OPENQA_LIBPATH} // OPENQA_LIBPATH, @INC);` then? `OPENQA_LIBPATH` is already supposed to be the default anyways.

And this way it would also override the default which is better. Because if you set the env variable incorrectly it would otherwise silently fallback to the packaged version and you would wonder why your changes have no effect.

Comment (foursixnine) [reply testapi.pm]: Not really: http://phobos.suse.de/tests/11075,

It will only complain when the parser cannot be found: 

Use of uninitialized value in require at (eval 130) line 1 [in the autoinst-log.txt](http://phobos.suse.de/tests/11074#)

Comment (coolo) [reply testapi.pm]: is this opposing @Martchus ? Because his suggestion makes sense to me

Comment (foursixnine) [reply testapi.pm]: @coolo Didin't see @Martchus comment :P, changing that... In any case, as it is, is not troublesome.

File: testapi.pm [hunk 2]
```diff
@@ -1694,12 +1694,12 @@ sub parse_extra_log {
 
     $file = upload_logs($file);
     my @tests;
-
+    use Data::Dumper;
```
Comment (coolo) [inline testapi.pm]: you slipped in unrelated changes

Comment (foursixnine) [reply testapi.pm]: Holy!

Comment (foursixnine) [reply testapi.pm]: Done xD
